src/routers/alpha-router/quoters/base-quoter.ts

```diff
@@ -41,10 +43,10 @@ import { GetQuotesResult, GetRoutesResult } from './model/results';
  */
 export abstract class BaseQuoter<
   CandidatePools extends
-    | V2CandidatePools
-    | V3CandidatePools
+    | SupportedCandidatePools
     | [V3CandidatePools, V2CandidatePools, CrossLiquidityCandidatePools],
```

Comment:
what about this line here? does it need updating with V4?

Reply:
We don't want to update this line yet, because it's touching the mixed routing/quoting.